### PR TITLE
Add amp and speaker scoring commands :flushed:

### PR DESCRIPTION
**Pull request by 5690Programmers**
Will merge after binding to buttons and testing

### CODE REVIEW

File: src/main/include/commands/ScoreAmpCommand.h [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    public:
+
+        enum class ScoreState {
+            FlywheelRamp,
+            Feeding,
+        };
+
+        explicit ScoreAmp(ScoringSubsystem* subsystem, IntakeSubsystem* intk) 
+            : m_score{subsystem}, m_intake{intk}, isFinished{false} {
+                AddRequirements(m_intake);
```
Comment (having11) [inline src/main/include/commands/ScoreAmpCommand.h:19]: Use initializer list instead of calling twice

**Review by having11 — CHANGES_REQUESTED** [review]
LBTM
disintegrate-1403807930

File: src/main/include/commands/ScoreSpeakerCommand.h [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+        void Execute() override {
+            if (m_state == ScoreState::FlywheelRamp) {
+                if (m_score->GetMotorAtScoringSpeed(ScoringDirection::SpeakerSide)) {
+                    m_score->SpinVectorSide(ScoringDirection::SpeakerSide);
+                    m_intake->In();
+                    m_state = ScoreState::Feeding;
+                }
+            } else if (m_state == ScoreState::Feeding) {
+                if (m_score->GetMotorFreeWheel(ScoringDirection::SpeakerSide)) {
```
Comment (nobody5050) [inline src/main/include/commands/ScoreSpeakerCommand.h:40]: same issue as above

**Review by nobody5050 — APPROVED** [review]

**Review by nobody5050 — COMMENTED** [review]
Make sure the note can’t get stuck

**Review by nobody5050 — APPROVED** [review]
LGTM!

**Pull request merged**